src/main/java/org/folio/helper/CheckinReceivePiecesHelper.java

```diff
@@ -472,9 +249,60 @@ private Future<Void> storeUpdatedPieceRecord(Piece piece, RequestContext request
       });
   }
 
-
-  protected List<String> getPoLineIdsForUpdatedPieces(Map<String, List<Piece>> piecesGroupedByPoLine) {
-    return EntryStream
+  /**
+   * @param pieceId piece id
+   * @return PO Line id corresponding to passed pieceId from request data
+   */
+  private String getPoLineIdByPieceId(String pieceId) {
+    return StreamEx
+      .ofKeys(piecesByLineId, values -> values.containsKey(pieceId))
+      .findFirst()
+      .orElse(EMPTY);
+  }
+
+  //-------------------------------------------------------------------------------------
+  /*
+  updateOrderAndPoLinesStatus
+   */
```

Comment:
these comments, also not useful,

some pattern can be used to divide methods by group, if this patter helpful here
```
getUser(){
  getName
}
getname(){}

postUser(){
 getUsername()
}
getUsername(){}

deleteUser(){}```

Reply:
Please review my comment regarding method grouping in this class